docs/userguides/quickstart.md

```diff
@@ -159,6 +159,16 @@ If you want to run specific files in a `scripts/` directory, you can do it with
 $ ape run deploy
 ```
 
+**Ape Logging:** 
+
+If you want to view more information while running any of the ape commands.
```

Comment:
Adjusting the log level does not necessarily cause there to be more information - that only really happens when setting to the `DEBUG` level (e.g. `--verbosity DEBUG`). But, you can also get _less_ information by setting the level to one that is less verbose (e.g. `--verbosity ERROR`). 99.9% of the time, I bet, `--verbosity` will be used to just enable debug logging though.

Reply:
I reworded the statement to incorporate this comment. Please let me know if I understand it properly.